app/javascript/components/public_health/PatientsTable.js

```diff
@@ -217,7 +217,7 @@ class PatientsTable extends React.Component {
         this.updateTable({ ...query, jurisdiction: jurisdictionId, page: 0 });
         this.updateAssignedUsers(jurisdictionId, this.state.query.scope, this.props.workflow, this.state.query.tab);
       }
-    } else if (event.target.name === 'assignedUser') {
+    } else if (event.target.name === 'assigned_user') {
```

Comment:
shouldn't this be `event.target.id` now because of this change? https://github.com/SaraAlert/SaraAlert/pull/450/files#diff-605db0c430cb0318fcd9986ece737d10d16c2ab6807a7c35737484322530bb49R503

Reply:
maybe something we can do as part of this refactor is change all the other inputs in the app as well from using `event.target.name` to `event.target.id`? @kylietmo @holmesie thoughts?

Reply:
you'd be correct - nice catch!

Reply:
I changed them all in this file for consistency - there's not many more that would need to be swapped

Reply:
The ones in `Exposure.js` and `MonitoringStatus.js` would require more of an overhaul so I will leave those for now

Reply:
aren't there just 2 (assigned user and jurisdiction id) in `Exposure.js`? I see that the logic in` MonitoringStatus.js` depends on both the `name` and `id` so I agree that it would require a bit more work

Reply:
in `Exposure.js` id="assigned_user" and id="jurisdiction_id" are used elsewhere so probably not best to tinker with that